script/lib/importer.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+import os
+
+BINARIES = {
+  'darwin': [
+    'libbookmarks_browser.a',
+    'libcomponent_metrics_proto.a',
+    'libfavicon_base.a',
+    'libgoogle_core_browser.a',
+    'libhistory_core_browser.a',
+    'libsearch_engines.a',
+    'libquery_parser.a',
+    'liburl_formatter.a',
+  ],
+  'linux': [
```

Comment:
have you checked these on linux and windows or are these a guess based on the mac libs?